sygnal/gcmpushkin.py

```diff
@@ -379,12 +403,12 @@ async def _dispatch_notification_unlimited(self, n, device, context):
             return failed
 
     @staticmethod
-    def _build_data(n, device):
+    def _build_data(n: Notification, device: Device) -> Dict:
```

Comment:
```suggestion
    def _build_data(n: Notification, device: Device) -> Dict[str, Any]:
```
(JSON-compatible dicts always have string keys, so this is usually a good enough type signature for us)

Reply:
> so this is usually a good enough type signature for us

Yes and no. The `Any` values really limit what the typechecker can spot and interrogate!